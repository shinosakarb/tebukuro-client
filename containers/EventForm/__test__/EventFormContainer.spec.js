import React from 'react'
import { shallow } from 'enzyme'
import ToJson from 'enzyme-to-json'
import { EventForm } from '../index'

jest.mock('../../../actions/Event')
//TODO: Remove virtual mode after API implemented.
jest.mock('../../../api/Event', () => {}, { virtual: true })

const props = {
  createEvent: jest.fn(),
}

describe('Event container', () => {
  it('should render self and subcomponents', () => {
    const wrapper = shallow(<EventForm {...props} />)
    const tree = ToJson(wrapper)

    expect(tree).toMatchSnapshot()
  })
})
