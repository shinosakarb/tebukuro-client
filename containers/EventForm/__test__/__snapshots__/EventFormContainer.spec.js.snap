// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Event container should render self and subcomponents 1`] = `
<div>
  <div>
    This is the EventForm Container.
  </div>
  <EventForm
    onSubmit={[Function]}
  />
</div>
`;
